src/kata5.reachDestination.js

```diff
@@ -1,3 +1,14 @@
-const reachDestination = (distance, speed) => {};
+const reachDestination = (distance, speed) => {
+  if (distance === 0 && speed === 0) {
+    return "I am here";
+  }
+  if (distance > 0 && speed === 0) {
+    return "The train has broken down";
+  }
+  const arrivalTime = distance / speed;
+  return `I will be there in ${Math.round(arrivalTime * 2) / 2} hours.`;
+};
```

Comment:
Minor formatting change could be done here, dont need to create the variable on line 8 for it only to be used once in the return statement. Leaving the blank line before the return statement is a convention for readability, not always needed for short functions though
```
const reachDestination = (distance, speed) => {
  if (distance === 0 && speed === 0) {
    return "I am here";
  }
  if (distance > 0 && speed === 0) {
    return "The train has broken down";
  }

  return `I will be there in ${Math.round((distance / speed) * 2) / 2} hours.`;
};
```